src/TestProblem/ELBDM/LSS/Init_TestProb_ELBDM_LSS.cpp

```diff
@@ -133,6 +190,61 @@ void SetParameter()
 
 } // FUNCTION : SetParameter
 
+//-------------------------------------------------------------------------------------------------------
+// Function    :  Flag_Region_LSS
+// Description :  Template for checking if the element (i,j,k) of the input patch is within
+//                the regions allowed to be refined
+//
+// Note        :  1. Invoked by Flag_Check() using the function pointer "Flag_Region_Ptr",
+//                   which must be set by a test problem initializer
+//                2. Enabled by the runtime option "OPT__FLAG_REGION"
+//
+// Parameter   :  i,j,k       : Indices of the target element in the patch ptr[0][lv][PID]
+//                lv          : Refinement level of the target patch
+//                PID         : ID of the target patch
+//
+// Return      :  "true/false"  if the input cell "is/is not" within the region allowed for refinement
+//-------------------------------------------------------------------------------------------------------
+bool Flag_Region_LSS( const int i, const int j, const int k, const int lv, const int PID )
+{
+
+   const double dh     = amr->dh[lv];                                         // cell size
+   const double Pos[3] = { amr->patch[0][lv][PID]->EdgeL[0] + (i+0.5)*dh,     // x,y,z position
+                           amr->patch[0][lv][PID]->EdgeL[1] + (j+0.5)*dh,
+                           amr->patch[0][lv][PID]->EdgeL[2] + (k+0.5)*dh  };
+
+   bool Within = true;
+
+// get the zoom-in ID that can collect the zoom-in box's volume and center based on the current simulation time
+   int zoomID; 
+   for (int s=0; s<ZoomIn_MaxSnap; s++)
+   {
+      if ( Time[0] <= ZoomIn_a[s] )   continue;
+      zoomID = s;
+      break;
+    } // for (int s=0; s<ZoomIn_MaxSnap; s++)
```

Comment:
[Style] Wrong indent:
```suggestion
   } // for (int s=0; s<ZoomIn_MaxSnap; s++)
```